<ui version="4.0" >
 <class>TabularCreateUi</class>
 <widget class="QDialog" name="TabularCreateUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="rowsL" >
     <property name="toolTip" >
      <string>Number of rows</string>
     </property>
     <property name="text" >
      <string>&amp;Rows:</string>
     </property>
     <property name="buddy" >
      <cstring>rowsSB</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>58</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="columnsL" >
     <property name="toolTip" >
      <string>Number of columns</string>
     </property>
     <property name="text" >
      <string>&amp;Columns:</string>
     </property>
     <property name="buddy" >
      <cstring>columnsSB</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QSpinBox" name="rowsSB" >
     <property name="toolTip" >
      <string>Number of rows</string>
     </property>
     <property name="buttonSymbols" >
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum" >
      <number>511</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSpinBox" name="columnsSB" >
     <property name="toolTip" >
      <string>Number of columns</string>
     </property>
     <property name="buttonSymbols" >
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum" >
      <number>511</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <tabstops>
  <tabstop>rowsSB</tabstop>
  <tabstop>columnsSB</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
